pkg/gadgets/networkpolicy/gadget.go

```diff
@@ -102,6 +103,11 @@ func (f *TraceFactory) Operations() map[string]gadgets.TraceOperation {
 }
 
 func (f *Trace) Start(trace *gadgetv1alpha1.Trace) {
+	if runtime.GOARCH == "arm64" {
```

Comment:
Maybe try the other way around: `if runtime.GOARCH != "amd64" {` ?

Reply:
This was updated for traceloop but not for networkpolicy. Reopening.